packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -55,9 +66,30 @@ export default function BookmarkFeedLayout({
       <FeedPageHeader className="mb-5">
         <h3 className="font-bold typo-callout">Bookmarks</h3>
       </FeedPageHeader>
-      <CustomFeedHeader className="relative mb-6">
+      <CustomFeedHeader className="flex mb-6">
         {searchChildren}
+
+        <Button
+          className="ml-4 btn-secondary hidden laptop:flex"
+          icon={<SourceIcon />}
+          onClick={() => setShowSharedBookmarks(true)}
+        >
+          Share bookmarks
+        </Button>
+        <Button
+          className="ml-4 btn-secondary flex laptop:hidden"
+          icon={<SourceIcon />}
+          onClick={() => setShowSharedBookmarks(true)}
+          iconOnly
+        />
```

Comment:
I think it would be better to store the props in this button in a variable and just spread it for maintainability purposes - we would only have to change in one place when needed.

Reply:
What about something like this ? 

```
const shareBookmarksButton = (style: string, text?: string) => (
    <Button
      className={style}
      icon={<SourceIcon />}
      onClick={() => setShowSharedBookmarks(true)}
    >
      {text}
    </Button>
  );

  return (
    ...
        {shareBookmarksButton(
          'hidden laptop:flex ml-4 btn-secondary',
          'Share bookmarks',
        )}
        {shareBookmarksButton('flex laptop:hidden ml-4 btn-secondary')}
...

```

Reply:
Great idea!